specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1975 @@
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Revalidates the encryption protector": {
+            "$ref": "./examples/WorkspaceManagedSqlServerEncryptionProtectorRevalidate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/serverKeys/{serverKeyName}": {
```

Comment:
This is missing the individual GET and collection GET

Reply:
For the server keys. SQL Team asks to proxy just Put and Delete Calls. Thus, didn't added the Get and List calls for them

Reply:
@pilor is any more action required from Kunal?

Reply:
Removed API related to Server Key. Feature moved inside another existing API